ports/raspberrypi/boards/archi/mpconfigboard.mk

```diff
@@ -0,0 +1,24 @@
+USB_VID = 0x1043
+USB_PID = 0x2E8A
```

Comment:
Where did you get this VID and PID? Are you the manufacturer of this board?

Reply:
Yes, I am part of the team that developed that board, that VID and PID are the ones that gave us the raspberrypi foundation for this board

Reply:
Aha, I see. But I think you've reversed the VID and PID. The RPi **VID** is 0x2E8A. (https://github.com/raspberrypi/usb-pid)

Reply:
Thanks, I hadn't noticed that. :)